src/uct/api/tl.h

```diff
@@ -59,6 +51,14 @@ typedef struct uct_iface_ops {
     int          (*iface_is_reachable)(uct_iface_h iface,
                                        const uct_iface_addr_t *addr);
 
+    void         (*ep_destroy)(uct_ep_h ep);
```

Comment:
I'd add some comment that function pointers in uct_iface_ops should be in strict order: all functions with iface as a first parameter must come first. Also some test with uct_base_iface_set_error function would be good to have, because it seem to be quite easy to introduce an error by adding some new function to uct_iface_ops in a wrong place.